Source/Plugins/Core/com.equella.core/src/com/tle/web/controls/youtube/YoutubeHandler.java

```diff
@@ -286,9 +287,10 @@ public void registered(String id, SectionTree tree) {
 
               YoutubeResultOption result = new YoutubeResultOption(videoId);
               result.setAuthor(new KeyLabel(ADD_AUTHOR_LABEL, vidInfo.getChannelTitle()));
-              result.setDate(
-                  dateRendererFactory.createDateRenderer(new Date(vidInfo.getPublishedAt())));
-
+              Optional<Date> parsedDate =
+                  YoutubeUtils.parseDateModifiedToDate(vidInfo.getPublishedAt());
+              parsedDate.ifPresent(
+                  date -> result.setDate(dateRendererFactory.createDateRenderer(date)));
```

Comment:
Similar to above.

```suggestion
              YoutubeUtils.parseDateModifiedToDate(vidInfo.getPublishedAt())
                .ifPresent(
                  date -> result.setDate(dateRendererFactory.createDateRenderer(date)));
```

(Unless I'm missing them being used somewhere else.)